Solutions/Corvus.Monitoring.AspnetCore.Mvc.Demo/Program.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="Program.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddControllersWithViews();
+
+// Add Corvus.Monitoring. This will add the "null" implementations of IOperationsInstrumentation and IExceptionsInstrumentation
+// by default.
+builder.Services.AddInstrumentation();
+
+WebApplication app = builder.Build();
```

Comment:
I _think_ that you're building a completely ordinary ASP.NET Core pipeline here, so the only unusual thing about this file, aside from the funky `partial` class at the bottom is that we called `AddInstrumentation` on the service.

I'd like a comment asserting this directly—I think the purpose of this Demo project it to show people how they're expected to use this feature, so it would be useful to make it completely clear that you just need that one additional DI setup call.